benchmark/Lyap_benchmark.jl

```diff
@@ -0,0 +1,37 @@
+#Lyapunov Benchmarks
+
+@taylorize function lorenz!(dx,x,param,t)
+    σ, ρ, β = param
+     dx[1] = σ*(x[2]-x[1])
+    dx[2] = x[1]*(ρ-x[3])-x[2]
+    dx[3] = x[1]*x[2]-β*x[3]
+    nothing
+end
+
+SUITE = BenchmarkGroup()
+
+    # const TI = TaylorIntegration
+    const _abstol = 1.0e-20
+    const _order = 28
+    const pars = [16.0, 45.92, 4.0]
+    const t0 = 0.0
+    const tf1 = 1000.0
+    const maxsteps1 = 500_000
+    const tf2 = 10_000.0
+    const maxsteps2 = 50_000_000
+    const tf3 = 100_000.0
+    const maxsteps3 = 500_000_000
+    q0 = [0.2, 0.0, 0.0, 3.0]
```

Comment:
Three comments in these lines:
- Please delete the 4 spaces (or tab), so everything is properly aligned
- `q0` is the initial condition, so it must have the same dimension as the equations of motion, i.e. 3. I'd suggest to use `q0 = [19.0, 20.0, 50.0]`, as in the tests. Also, perhaps having it as `const` is ok.
- You need to add the line `set_variables("δ", order=1, numvars=3);", so the stability matrix can be computed using automatic differentiation.